### PR TITLE
Microsoft.Extensions.FileSystemGlobbing PACKAGE.md

### PR DESCRIPTION
**Pull request by EnsignPayton**
Fill out details for `Microsoft.Extensions.FileSystemGlobbing` NuGet package readme

See https://github.com/dotnet/runtime/pull/92228

### CODE REVIEW

**Comment by EnsignPayton** [comment]
@dotnet-policy-service agree

File: src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/PACKAGE.md [hunk 1]
```diff
@@ -2,40 +2,48 @@
...
 <!-- Links to further documentation. Remove conceptual documentation if not available for the library. -->
 
-* [Conceptual documentation](https://learn.microsoft.com/en-us/dotnet/standard/serialization/**LIBRARYNAME**/overview)
-* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/**LIBRARYNAME**)
-
-## Related Packages
-
-<!-- The related packages associated with this package -->
+* [Conceptual documentation](https://learn.microsoft.com/en-us/dotnet/core/extensions/file-globbing)
+* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.filesystemglobbing)
```
Comment (ViktorHofer) [inline src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/PACKAGE.md:46]:
```suggestion
* [API documentation](https://learn.microsoft.com/dotnet/api/microsoft.extensions.filesystemglobbing)
```

**Review by ViktorHofer — APPROVED** [review]
Thanks a lot 👍

File: src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/PACKAGE.md [hunk 2]
```diff
@@ -2,40 +2,48 @@
...
 <!-- The main types provided in this library -->
 
 The main types provided by this library are:
 
-* ``
-* ``
-* ``
+* `Microsoft.Extensions.FileSystemGlobbing.Matcher`
 
 ## Addtional Documentation
```
Comment (ViktorHofer) [inline src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/PACKAGE.md:41]:
```suggestion
## Additional Documentation
```

**Pull request merged**